packages/@aws-cdk/aws-ecs/test/test.container-definition.ts

```diff
@@ -804,20 +804,6 @@ export = {
         ],
       }));
 
-      test.done();
-    },
-    'throws when using environment files for a Fargate task'(test: Test) {
-      // GIVEN
-      const stack = new cdk.Stack();
-      const taskDefinition = new ecs.FargateTaskDefinition(stack, 'TaskDef');
-
-      // THEN
-      test.throws(() => taskDefinition.addContainer('cont', {
-        image: ecs.ContainerImage.fromRegistry('test'),
-        memoryLimitMiB: 1024,
-        environmentFiles: [ecs.EnvironmentFile.fromAsset(path.join(__dirname, 'demo-envfiles/test-envfile.env'))],
-      }), /Cannot specify environment files for a task using the FARGATE launch type in container/);
-
       test.done();
     },
```

Comment:
Instead of deleting the test, can you update the test to succeed?

Reply:
I'm not sure I agree. Why should we test for environment files if they are no now permitted?

Reply:
We should verify that it's being added to the template correctly now that it's permitted

Reply:
The previous test ([Line 785](https://github.com/aws/aws-cdk/blob/b95181b404cf17846082a3eef7239780b75d7916/packages/%40aws-cdk/aws-ecs/test/test.container-definition.ts#L785)) already covers the expected behaviour.

Reply:
The previous test is for a Ec2TaskDefinition, this would be for a FargateTaskDefinition.

Reply:
+1 here. Let's make sure we cover this (success) scenario with a test.